utmp has ut_time member, not a ut_tv struct member

--- origsrc/gnome-control-center-3.0.1.1/panels/datetime/tz.c	2011-04-15 17:33:03.000000000 -0500
+++ src/gnome-control-center-3.0.1.1/panels/datetime/tz.c	2011-05-23 17:54:12.753282300 -0500
@@ -217,7 +217,7 @@ tz_info_from_location (TzLocation *loc)
 	curtime = time (NULL);
 	curzone = localtime (&curtime);
 
-#ifndef __sun
+#if !defined(__sun) && !defined(__CYGWIN__)
 	/* Currently this solution doesnt seem to work - I get that */
 	/* America/Phoenix uses daylight savings, which is wrong    */
 	tzinfo->tzname_normal = g_strdup (curzone->tm_zone);
